Server-client discovery use-case [7789] (#95)
eProsima/Fast-RTPS-docs: docs/use-cases.rst

|                                   | eliminating all EDP meta traffic.                                               |
+-----------------------------------+---------------------------------------------------------------------------------+

.. START SUBSEC:DISCOVERY-SERVER

.. _server-client-discovery-use-case:

Server-Client Discovery
-----------------------

Considering a scenario in which a large number of communication agents, called participants in this case, are deployed,

> an alternative to the default SIMPLE discovery mechanism of the RTPS standard may be used. For this purpose, Fast RTPS

Reviewer: ```suggestion
an alternative to the default RTPS standard SIMPLE discovery mechanism may be used. For this purpose, Fast-RTPS
```

> provides for the client-server discovery mechanism, in which a server participant operates as the central point of

Reviewer: ```suggestion
provides a client-server discovery mechanism, in which a server participant operates as the central point of
```

communication; the server collects and processes the metatraffic sent by the client participants, and distributes the

> appropriate information to the previous participants.

Reviewer: ```suggestion
communication, that is the server collects and processes the metatraffic sent by the client participants, and distributes the
appropriate information among the rest of the clients.
```

> discovery phases, and distributes the appropriate information to the previous participants.

Reviewer: ```suggestion
communication; the server collects and processes the metatraffic sent by the client participants, and distributes the appropriate information to the previous participants.
```

Various discovery server use cases are presented below.

.. _discovery_server_major_scenario_setup:

Discovery-Server major scenario setup

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: I think this is a UDPv4 example. I'd call it something on those lines
